### PR TITLE
Proofreading: Minor language issues

### PR DESCRIPTION
**Pull request by MakisH**
A significant part of these issues is use of comma, in particularly the [Oxford/serial comma](https://en.wikipedia.org/wiki/Serial_comma), and the common issue between `blah, dass` and `blah that`.

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -898,8 +899,8 @@ as is a range of different roles from King's Digital Lab at King's College Londo
 What we have defined above are intended to be base skills that an RSE irrespective of domain, place, and time should know about.
 But not all RSEs are created equal, they specialise in different areas,
 some of which we want to present below. Many of the specialisations may overlap,
-so the same RSE might for example work on data management and Open Science.
-We categorise them into those, that can be viewed as a specialisation within RSE-specific topics,
+so the same RSE might for example work on Data Management and Open Science.
```
Comment (MakisH) [inline competencies.md:902]: Not sure what the rule is here regarding capitalization, but I guessed that there is a preference for Upper Case.

File: competencies.md [hunk 2]
```diff
@@ -316,16 +316,16 @@ The role of an RSE lies somewhere on the spectrum between that of a researcher
 (the "R") and a software engineer (the "SE") and, therefore, requires
 competencies in both fields. RSEs typically apply their knowledge and
 experience in larger teams which allows them to cultivate this hybrid nature.
-Therefore, we categorise the competencies into software engineering skills,
-research skills, and team skills with a particular focus on the software and
+Therefore, we categorise the competencies into three categories: *software engineering skills*,
```
Comment (MakisH) [inline competencies.md:319]: This is mostly a hint to the subsections structure.

File: competencies.md [hunk 3]
```diff
@@ -1092,15 +1092,15 @@ This paper started from a community workshop at deRSE23 in Paderborn
...
 One outcome of this diverse gathering is that RSEs from far away fields gather
-around similar core concepts but at the same time share a vision of how to
+around similar core concepts, and at the same time share a vision of how to
 update the scientific ecosystem to the age of digitalization.
 In this publication, we have tried to formalise these concepts.
 We have formulated a set of values that guide our actions in society,
-manifestly making RSEs part of the scientific community that shares the ideals of good scientific practice,
-but at the same time,
+manifestly making RSEs part of the scientific community that shares the ideals of good scientific practice.
+At the same time,
```
Comment (MakisH) [inline competencies.md:1100]: This sentence is already too complex and too long.

File: competencies.md (hunk 1 above)
Comment (jngrad) [reply competencies.md:902]: I have mixed feelings about capitalization. For example, in the intro we have capitalized "Machine Learning", which makes it stand out, even though it is not the main focus of the paper. More generally, I often ask myself if there is a need to capitalize "Molecular Dynamics" or "Open Source" in my talk/poster/paper abstracts, especially when there is no corresponding acronym to justify the uppercase. Here I would have personally went for "data management and open science", but it's probably just a matter of preferences.

**Review by jngrad — APPROVED** [review]

File: competencies.md (hunk 1 above)
Comment (MakisH) [reply competencies.md:902]: I have the same feeling and preference as you, actually. I am fine with any consistent style.

Comment (CaptainSifff) [reply competencies.md:902]: most likely it's just due to having a bunch of germans on the paper....

Comment (CaptainSifff) [reply competencies.md:902]: and the original at least is inconsistent here...

Comment (CaptainSifff) [reply competencies.md:902]:
```suggestion
so the same RSE might for example work on data management and open science.
```

**Review by CaptainSifff — APPROVED** [review]

File: .github/spellignore.txt [hunk 4]
```diff
@@ -62,7 +62,7 @@ findability
 findable
 FIXME
 Forschung
-FORTRAN
+Fortran
```
Comment (CaptainSifff) [inline .github/spellignore.txt:65]: @MakisH, @BeastyBlacksmith , @jngrad  do we know, why the spellchecker doesn't pick up this change?

Comment (jngrad) [reply .github/spellignore.txt:65]: I tried a few tweaks, but the action flat out refuses the modern spelling of "Fortran". After adding it to our ever-growing sed expression in b169425fc7e32, the spellchecker seems satisfied.

**Pull request merged**